src/views/Home/components/Mission/mission.scss

```diff
@@ -1,3 +1,108 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/responsiveness";
+@use "~stylesheets/typography";
+@use "~stylesheets/variables";
 
-/* Delete these lines when you begin development */
+.Mission {
+  position: relative;
+
+  /* Text Layer */
+  &--text {
```

Comment:
we should have a `max-width` to constrain how wide the paragraph renders at larger screens.. I noticed adding a `max-width` here doesn't apply to the second mission section in the same way, so the implementation is worth revisiting.. the DOM structure is worth applying here to keep the widths uniform across sections 
![Screenshot from 2020-12-19 13-37-18](https://user-images.githubusercontent.com/19658882/102700169-f2bef980-41ff-11eb-93b9-a7b22661400e.png)

Reply:
I'm not too sure why you're seeing it like that - this is how my view looks like on a large display. Can you check on it again?

![image](https://user-images.githubusercontent.com/33771623/103041457-506a8300-452b-11eb-82b2-3c095429a164.png)

Reply:
this is the case I'm referring to: the text containers aren't bounded, and the paragraphs turn into one long sentence
![Screenshot from 2020-12-23 16-36-04](https://user-images.githubusercontent.com/19658882/103046474-4782ad00-453d-11eb-85bf-16b0409cb70f.png)

Reply:
this is what the hero-in-progress & milestones looks like at the same resolution since those components have width constraints on the containers
![Screenshot from 2020-12-23 16-42-44](https://user-images.githubusercontent.com/19658882/103046654-e9a29500-453d-11eb-9745-3f4dabc54e8a.png)

Reply:
Gotcha, I think I resolved it - let me know how it looks!